doc/rtd/explanation/format.rst

```diff
@@ -3,29 +3,29 @@
 User data formats
 *****************
 
-User data that will be acted upon by ``cloud-init`` must be in one of the
-following types.
+User data is opaque configuration data provided by a platform to an instance at
+launch to perform supplemental configuration operations on that instance at
+first boot. User data that will be acted upon by ``cloud-init`` must be in one
+of the following types.
```

Comment:
This feels more verbose than it needs to be. I think that we could probably strike some words to be more succinct without loosing the important bits:

```suggestion
User data is configuration data provided by a platform to an instance at
launch to configure the instance. User data can be one of the following types.
```

Reply:
Who'd have though less words, more clear meaning :). Thanks for these suggestions.